hardware-api.md

```diff
@@ -91,7 +97,23 @@ pin.analogRead(function(error, value) {
 
 ### PWM pins
 
-PWM pins are not yet implemented. See [#21](https://github.com/tessel/t2-firmware/issues/21).
+**PWM pins** are pulse-width modulated pins ([wiki link](http://en.wikipedia.org/wiki/Pulse-width_modulation)). Essentially, PWM is a digital signal that spends between 0% and 100% of its time on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
+
+Tessel has four PWM pins: TX and RX (pins 5 and 6) on each module port.
+
+Here is an example of setting a PWM pin:
+```js
+var tessel = require('tessel'); // import tessel
+var port = tessel.port.A; // select one of the two ports
+var myPin = port.pwm[0]; // equivalent to port.digital[0]. Can also be port.pwm[1] or port.digital[1]
+// Tell the Tessel that the frequency of its pwm pins is 980 Hz
+tessel.pwmFrequency(980);
+// Set duty cycle
+myPin.pwmDutyCycle(0.6); // set the pin to be on 60% of the time
+```
+
+Note: the `pwmFrequency` function *must* be called before `pwmDutyCycle` and that re-setting
+`pwmFrequency` will disable PWM output until `pwmDutyCycle` is called again. Additionally, `pwmFrequency` is capable of frequencies from 1Hz to 5kHz. 
```

Comment:
Scratch "additionally", it's a separate point. Maybe it should be its own separate note.